iModelCore/libsrc/openssl/BeOpenSSL.mke

```diff
@@ -33,6 +33,8 @@ vendorSrc = $(_MakeFilePath)vendor/
 
 %if defined(iTwinNativeThirdParty)
   appName = BeOpenSSL
+  always:
+    $(BBPYTHONCMD) $(bsiScripts)RegexReplaceInFile.py iTwinOpenSSL BeOpenSSLB02 $(_MakeFilePath)BeOpenSSL.def

```

Comment:
Modifying the source tree during the build is a bad idea. It will make everyone's source appear modified. Copy the file to the output tree (not symlink) and use it from there. Modify as needed.

Reply:
ok let me do that

Reply:
I have now modified the logic accordingly.